SRC/Portfolio-rebuild/portfolio.css

```diff
@@ -0,0 +1,80 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border: none;
+  background: none;
+}
+.mainContainer1{
+  height: 100vh;
+  width: 100%;
+  border: 1px solid blue;
+  /* background-image: url(https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.ca%2Fpin%2F181129216234773932%2F&psig=AOvVaw2JN-y8ZM0cUWhS_SDIB7TT&ust=1581646622734000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCICFz7G7zecCFQAAAAAdAAAAABAK); */
+}
+header{
+  height: 5%;
+  /* border: 1px solid red; */
+  background-color:black;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
```

Comment:
add some padding so the text doesn't touch the top of the page: `padding-top: 10px;`